build.sh

```diff
@@ -2,7 +2,7 @@
 ## START STANDARD SITE BUILD SCRIPT INCLUDE
 readonly THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
 readonly BOOTSTRAP="$(dirname "$THIS_SCRIPT")/resources/bootstrap.inc.sh"
-readonly BOOTSTRAP_VERSION=v0.10
+readonly BOOTSTRAP_VERSION=chore/v0.11
```

Comment:
This is for engineering testing until keymanapp/shared-sites#34 gets merged.
```suggestion
readonly BOOTSTRAP_VERSION=v0.11
```